index.js

```diff
@@ -892,7 +897,9 @@ class Batch {
     this.blocks.clear()
     this.length = 0
 
-    return this._appendBatch(batch)
+    await this._appendBatch(batch)
+
+    return this.close()
```

Comment:
This also fixes a potential bug that Hans pointed out, it closes this:
https://github.com/holepunchto/hyperbee/blob/b527eeafa7f2f83f9af2f62516be8ed17596391d/index.js#L843

Reply:
Its never a snap when appending